flight/PiOS/STM32F4xx/pios_flash_internal.c

```diff
@@ -34,6 +36,9 @@
 #include "pios_semaphore.h"
 #include <stdbool.h>
 
+/* Todo: consider relocating to a good header */
+bool __attribute__((weak)) vehicle_is_armed = false;
```

Comment:
Oh, you mean a direct cross-module call from manualcontrol -> flash? But f4 flash internal isn't present on all targets, so we could have a weak call symbol or..